### DISCUSSION TITLE
What's the purpose of the chardonnay runner?

### DISCUSSION THREAD

**Discussion by bound-variable** [bottlesdevs/Bottles#721] [General]
I can't find any info on this runner.

Thanks

**Comment by mirkobrombin** [comment]
Hi, we are deprecating this section for forums.usebottles.com

Btw Chardonay was the name of our Vaniglia runner, before the rename. More info [here](https://github.com/bottlesdevs/Bottles/issues/356), [here](https://docs.usebottles.com/components/runners) and [here](https://github.com/bottlesdevs/build-tools/tree/main/runners/vaniglia#readme).

Please consider posting this in the new forum so that I can reply to other users as well.
